pom.xml

```diff
@@ -236,6 +237,17 @@ THE SOFTWARE.
         </dependency>
     </dependencies>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.amazonaws</groupId>
+                <artifactId>aws-java-sdk-bom</artifactId>
+                <version>${aws-java-sdk.vesion}</version>
```

Comment:
There's a typo here, and I'm not sure this dependencyManagement section is needed at all.

Reply:
Ah, I see the typo. I'll fix it.

Regarding the dependencyManagement, it's a trick I learned from work. There's something weird with the `aws-java-sdk` (which is a transitive dependency through the `org.jenkins-ci.plugins:aws-java-sdk`) maven dependency that causes maven to (in some unknown circumstances) download a lot of old versions of the various dependencies. I.e, if you have version `1.11.562` of the dependency maven will still download e.g version `1.11.4xx` and a bunch of other versions. On an empty m2 repository this takes a loooot of time.

For some reason, adding the bom eliminates this problem. I have no idea why, and I have no idea why not everyone has the problem.

I can remove it if you want, but it shouldn't do any harm and it will reduce initial build time for some.

Reply:
```suggestion
                <version>${aws-java-sdk.version}</version>
```